miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -250,6 +254,15 @@ open class MiniAppMessageBridge {
                 object : TypeToken<DevicePermission>() {}.type
             )
 
+            if (permissionParam.permission == MiniAppDevicePermissionType.LOCATION.name) {
```

Comment:
Actually, When I checked the JS-MiniApp code then I found that the `MiniAppMessageBridge.onRequestDevicePermission()` is the entry point to check for the permissions. So I thought before going any further it should first check whether the location service is enabled or not. Even if the location service is disabled after granting the permission it will still check for location enable case.

Yes, but if in case if it's surpass the `MiniAppMessageBridge.onRequestDevicePermission` and directly call the `WebChromeClient` methods then, in that case it won't check. 

So to address that case I've moved the checking at the `WebChromeClient.onGeolocationPermissionsShowPrompt` so it'll check for all the cases.